src/main/java/org/apache/commons/text/lookup/DefaultStringLookup.java

```diff
@@ -117,14 +115,6 @@ public enum DefaultStringLookup {
      */
     XML(StringLookupFactory.KEY_XML, StringLookupFactory.INSTANCE.xmlStringLookup());
 
-    /**
```

Comment:
This is handy, why remote it?

Reply:
I don't believe it is appropriate to include a `main` method here. If the values are not easily accessible via the documentation, then that indicates a failure in the documentation and not a need for a one-off tool.